### PR TITLE
Make community libraries clearer.

### PR DESCRIPTION
**Pull request by japgolly**
Slight tidying up of the Libraries section.
The test-libs page now has less repetitive language and separates the libs from the frameworks. The lib-group index now looks much, much clearer.

### CODE REVIEW

**Comment by ochrons** [comment]
LGTM, needs to be rebased.

**Comment by japgolly** [comment]
Updated & rebased.

File: _data/library/testlibs.yml [hunk 1]
```diff
@@ -28,3 +20,14 @@
       url: https://github.com/greencatsoft/greenlight
       desc: BDD style testing framework for Scala and Scala.js.
       dep: '"com.greencatsoft" %%% "greenlight" % "0.2"'
+
+- topic: Property testing
```
Comment (sjrd) [inline _data/library/testlibs.yml]: Is it OK to have a blank line in the middle of the list in YAML?

Comment (japgolly) [inline _data/library/testlibs.yml]: Yeah it's fine. As to the "why", it just makes it a bit easier to see.

**Comment by sjrd** [comment]
LGTM

**Pull request merged**